app/auth/models.py

```diff
@@ -10,8 +10,9 @@
 
 
 class JWT(object):
```

Comment:
@rufuspollock This can be done. As the flows for getting toke for site and dpmpy are  different. 
The site gets the token from API `{api}/auth/callback` and client gets the token from API `{api}/auth/token`. We can set different expiry time.